yang.go

```diff
@@ -146,6 +148,15 @@ Formats:
 		stop(0)
 	}
 
+	for _, path := range paths {
+		paths, err := yang.PathsWithModules(path)
```

Comment:
FWIW: the "shadow" linter will be happier if you used a different name than `paths`, as you're shadowing the `paths` variable declared on line 94.